include/deal.II/lac/solver_cg.h

```diff
@@ -166,6 +182,33 @@ class SolverCG : public Solver<VECTOR>
          const VECTOR         &b,
          const PRECONDITIONER &precondition);
 
+  /**
+   * Connect a slot to retrieve the CG coefficients. The slot will be called
+   * with alpha as the first argument and with beta as the second argument.
+   * Called once per iteration.
+   */
+  boost::signals2::connection
+  connect_coefficients_slot(
+    const std_cxx11::function<void (double,double)> &slot);
+
+  /**
+   * Connect a slot to retrieve the estimated condition number.
+   * Called on each iteration if every_iteration=true otherwise called
+   * after all iterations.
```

Comment:
Good point. How about this as a replacement text: "...otherwise called once when iterations are ended (i.e., either because convergence has been achieved, or because divergence has been detected)."?